docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,307 @@
+the section named Specification.
+
+  * Make sure that another peer will be able to implement a feature/requirement.
+
+Blueprints are not for you; they are for the community. If you know that a
+problem exists and know how to fix it, the most natural course of action would
+be to start coding this fix and submitting a Pull Request. While this is still
+valid for most of the cases, some important architectural changes must be
+discussed first to explain the “why”, “what” and “how” to keep everyone on the
+same page, avoid unnecessary coding, and most importantly: allow others to
+implement it in case you are not available.
+
+  * Improve the code review quality and time.
+
+Having a better understanding of the problem and the big picture is better for
+code review. It is harder to capture that from the Pull Request (PR)
+description and PR changes. Developers that are aware of the problem tend to
+review your changes with the problem in mind and quickly. 
+
+  * Reduce onboarding time for new members.
+
+Having a history of how we made an architectural decision and why we are
+implementing it this way will give new members reading material to understand
+our project, avoiding unnecessary discussions and meetings to explain
+something.
+
+  * Create a common standard that will make it easier for readers.
+
+With an open RFC, authors tend to organize the ideas in different ways with
+different sections. Having a very well structured document with common sections
+will make it easier for readers to understand the problem and the solution.
+
+  * Track the progress of a significant implementation.
+
+We could use the blueprints header “status” line to track the progress of some
+features. We could even have a page parsing and listing all the blueprints with
+the title, author, status, and target milestone for that feature.
+
+  * Find the middle ground between an “overthinking” and “auto-pilot.”
+
+Last but not least: we are *not trying* to overthink here and/or slow down our
+development processes. The idea is to have a lightweight document, very
+objective that will save us time in the medium and long run. We don’t need to
+overthink trying to find any possible scenario that we don’t have a use case.
+But we should also avoid the “auto-pilot” mode to fix things.
+
+
+Specification
+*************
+
+One blueprint per topic
+-----------------------
+
+Try to follow the minimalist approach and be concise with content relevant to
+one particular topic. If you have a more general topic to discuss, you should
+set the type as “Epic Blueprint” (more below) but still try to be concise and
+focused on the subject.  File format and location
+
+Our current documentation already uses ReStructuredText (.rst) format, so we
+will adopt .rst format here too. All blueprints will be located inside
+`docs/source/blueprints` with the filename `BPXXX.rst`, where XXX is the number
+of the blueprint. Just pick the next number available for your blueprint.
```

Comment:
Hopefully no concurrent problems of multiple people starting a blueprint at the same time but probably it won't happen in practice.

Reply:
Yes, I thought about it and decided to go with this simple approach.

If we have this, both are going to be in draft mode, and we can request changes before the merge.